src/App.js

```diff
@@ -161,9 +166,12 @@ export default class App extends React.PureComponent {
             return <AppLoading />;
         }
 
+        // FIXME: memoize this
```

Comment:
Please do in this PR

Reply:
Can't do right now. We need to first convert App.js to functional component!